images/ubuntu/scripts/tests/Browsers.Tests.ps1

```diff
@@ -1,12 +1,11 @@
-Describe "Firefox" -Skip:((-not (Test-IsUbuntu20)) -and (-not (Test-IsUbuntu22))) {
+Describe "Firefox" 
     It "Firefox" {
         "firefox --version" | Should -ReturnZeroExitCode
     }
 
     It "Geckodriver" {
         "geckodriver --version" | Should -ReturnZeroExitCode
     }
-}
```

Comment:
The pester test structure is broken.  The `Describe` section should be surrounded by `{ }` characters.
e.g.  
```
Describe "pack of tests" {
    It "sometest" {
        ... some code ...
    }
}